apps/common/controllers/add-another-address-loop.js

```diff
@@ -35,16 +12,17 @@ module.exports = class AddAnotherAddressController extends BaseController {
   }
 
   removeItem(req, res) {
-    const items = req.sessionModel.get(`${this.field}Addresses`);
-    req.sessionModel.set(`${this.field}Addresses`, _.omit(items, req.params.id));
-    const step = _.size(items) > 1 ? `/${this.field}-add-another-address` : `/${this.field}-postcode`;
+    const items = req.sessionModel.get(`${this.options.locals.field}Addresses`);
+    req.sessionModel.set(`${this.options.locals.field}Addresses`, _.omit(items, req.params.id));
```

Comment:
This `${this.options.locals.field}Addresses` is fugly, and done twice in this method, and at least one other method. And `${this.options.locals.field}-postcode`. 
My feeling is that you should have a helper method that creates these strings.